README.md

```diff
@@ -13,9 +22,6 @@ If you want to build `csaf-validator` on your own, please install Rust (see http
 git submodule init
 git submodule update --remote
 
-# make sure that local assets are in sync with git submodules
```

Comment:
Isn't this still relevant?

Reply:
Well, in that case a remote update of the submodules (lines above) wouldn't make sense either, since the two go together.
Question is whether you want to use the very latest version of submodules and their corresponding assets or not. I would say we want developers to always use the latest versions, and then this should stay in there.